stories/theme.AG_.mdx

```diff
@@ -19,89 +20,127 @@ taxonomy:
 
 <Block type="wide">
   <Figure>
-    <Embed height="405" src="https://www.youtube.com/embed/YbEV6vA-xgM" title="NASA and Agriculture" />
-    <Caption
-      attrAuthor='NASA'
-      attrUrl='https://nasa.gov/'
-    >
-    Learn how NASA data is being turned into tools to help farmers manage their crops in a changing climate.
+    <Embed
+      height="405"
+      src="https://www.youtube.com/embed/YbEV6vA-xgM"
+      title="NASA and Agriculture"
+    />
+    <Caption attrAuthor="NASA" attrUrl="https://nasa.gov/">
+      Learn how NASA data is being turned into tools to help farmers manage
+      their crops in a changing climate.
     </Caption>
   </Figure>
 </Block>
 
 <Block type="wide">
   <Prose>
   ## Agriculture
-  Producing food has always been challenging, and in the 21st century, human-caused climate change is already affecting food security through increasing temperatures, the frequency of extreme events, and changing precipitation patterns.  
+  Producing food has always been challenging, and in the 21st century, human-caused climate change is already affecting food security through increasing temperatures, the frequency of extreme events, and changing precipitation patterns.
+
+In the satellite era, Earth-observing data has increasingly become part of the food farming process. With observations from space and aircraft, combined with high-end computer modeling, NASA scientists work with partner agencies, organizations, farmers, ranchers, fishermen, and decision makers to share our understanding of the relationship between the Earth system and the environments that provide us food.
 
-  In the satellite era, Earth-observing data has increasingly become part of the food farming process. With observations from space and aircraft, combined with high-end computer modeling, NASA scientists work with partner agencies, organizations, farmers, ranchers, fishermen, and decision makers to share our understanding of the relationship between the Earth system and the environments that provide us food.
   </Prose>
 </Block>
 
 <Block>
   <Prose>
-    Water vapor moves around the world as ‘atmospheric rivers.’ Like regular rivers, they are long and narrow and transport water. In this visualization, the blue color shows a 3-D view of water vapor leaving the tropics and traveling to the U.S. West Coast.  These atmospheric rivers that begin in Hawai’i are named “pineapple express.”    
+    Water vapor moves around the world as ‘atmospheric rivers.’ Like regular
+    rivers, they are long and narrow and transport water. In this visualization,
```

Comment:
Are you using an auto-styler or did you add the indentations yourself? Just curious-- I hope you didn't spend too much time reviewing my .mdx!!

Reply:
Yeah, it is just an automatic linter so nw!